core/src/main/java/jenkins/security/RSAConfidentialKey.java

```diff
@@ -0,0 +1,84 @@
+package jenkins.security;
```

Comment:
It does default to the MIT header, but it has been indeed customary to add the copyright header explicitly.